x-pack/test/functional/apps/endpoint/index.ts

```diff
@@ -8,6 +8,7 @@ import { FtrProviderContext } from '../../ftr_provider_context';
 export default function({ loadTestFile }: FtrProviderContext) {
   describe('endpoint', function() {
     this.tags('ciGroup7');
+    this.tags('endpoint');
```

Comment:
You don't need the tag @EricDavisX .  You can already run just the endpoint test by using the describe title.  Just run `node ../scripts/functional_test_runner --grep endpoint`.

Reply:
ah, ok - thanks.  is it useful to leave it in all the same - in case anyone tries it before realizing the grep would work?  it doesn't hurt to have the tag I figure.